samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -151,14 +150,11 @@ private void RemoveArcGISMapView()
         }
     }
 
-    private void SceneChanged()
+    // Stop the menu background video from player as soon as other scene is loaded
+    private void StopVideo()
```

Comment:
You can get rid of `StopVideo()` since its not public and only one other place  that uses this method, moving lines `156-157` into line `111`

Reply:
Would it be better to use this as a method though for better organization purposes?

Reply:
I can see why you want to keep it, its to have a definitive Stop method to match the Start method.

So if the method was being used in at least two places, I would say yes keep it. 
But if its a. only used once, and b. a very short method (in this case only two lines), I would say no, the less methods the better.

However, I don't mind if this method lives, it's really a minor thing